package.json

```diff
@@ -28,6 +28,16 @@
       "url": "https://github.com/ApoorvSaxena/lozad.js/blob/master/LICENSE"
     }
   ],
+  "files": [
+    "dist/",
```

Comment:
Actually only the dist should be whitelisted. Otherwise all this others diles will be downloaded every time that someone download lozad from npm. That's only a wast of resources because only the dist files are used when somene require or imports the module.

Reply:
agree, fixed